x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -152,8 +152,9 @@ const AlertContextMenuComponent: React.FC<AlertContextMenuProps> = ({
       refetchQuery([timelineQuery]);
     } else {
       refetchQuery(globalQuery);
-      if (refetch) refetch();
     }
+
+    if (refetch) refetch();
```

Comment:
Could you add a comment here, mentioning that this is a temporary workaround?

Reply:
So, I was thinking that this is not a temporary workaround but `refetch` should always be outside. When someone works on https://github.com/elastic/security-team/issues/9598, they will need to take care that, refetch does not run twice. I have mentioned this comment in the above mentioned ticket.

What do you think?